include/materials/tensor.tcc

```diff
@@ -0,0 +1,226 @@
+//! Compute mean stress p
+inline double mpm::tensor::compute_mean_p(
+    const Eigen::Matrix<double, 6, 1>& stress) {
+
+  // Compute mean p in tension positive
+  double mean_p = 1. / 3. * (stress(0) + stress(1) + stress(2));
```

Comment:
return this instead of creating a variable.

Reply:
Noted